docs/source/development/tools/xtce_generator.rst

```diff
@@ -69,9 +78,9 @@ Follow the format below.
 Generating Telemetry XML Files
 -------------------------------
 
-This code block generates telemetry XML files for the packets defined in the
-`packets` dictionary. The telemetry XML files are saved in the directory specified
-by the `packet_definition_path` variable.
+Once you have your xtce processing file defined, you can run it with the
+following command:
+    python xtce_generator_instrument_name.py
```

Comment:
Put this in a code block as well

Reply:
Both of those code blocks also throw a warning when I ran `sphinx-build . build`:

```
/Users/maha7656/Documents/code/imap/imap_processing/docs/source/development/tools/xtce_generator.rst:32: ERROR: Unexpected indentation.
/Users/maha7656/Documents/code/imap/imap_processing/docs/source/development/tools/xtce_generator.rst:83: ERROR: Unexpected indentation.
```
So make sure you fix that before merging